documentation/source/building.rst

```diff
@@ -67,6 +67,10 @@ and
 
       Any arguments or flags to pass to the execution of the compiled simulation.
 
+.. make:var:: RUN_ARGS
+
+      Any argument to be passed to the "first" invocation of a simulator that runs via a TCL script. Used by Aldec Riviera-PRO and Mentor Graphics Questa simulator.
```

Comment:
- Riviera isn't currently using this variable. Either remove it from the docs, or add it to the Riviera Makefile.
- Please use one sentence per line, i.e. insert a newline before `Used`.
- Can you please document an example of when this might be useful, as you did in the commit message?